app/components/QuoteForm.tsx

```diff
@@ -23,14 +38,45 @@ export default function QuoteForm({
   field2Placeholder,
   field3Placeholder,
   submitButtonText,
+  setSubmitSuccessful,
 }: QuoteFormProps) {
-  const handleSubmit = (event) => {
-    event.preventDefault();
-    // Form submission logic goes here
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormData>();
+
+  const toast = useToast();
+
+  // EVENT HANDLERS
+  const onSubmit = (data: FormData) => {
+    console.log("data: ", data);
+    sendEmail(data);
+    // .then(response => {
+    //   if (response.success) {
+    //     setSubmitSuccessful(true);
+    //     reset();
+    //     toast({
+    //       title: "Success",
+    //       description: response.message,
+    //       status: "success",
+    //       duration: 5000,
+    //       isClosable: true,
+    //     });
+    //   } else {
+    //     toast({
+    //       title: "Failed",
+    //       description: response.message,
+    //       status: "error",
+    //       duration: 5000,
+    //       isClosable: true,
+    //     });
+    //   }
+    // });
```

Comment:
I removed the pop up alert for success (but kept it in for failure) and matched the success message to the Figma design - though it's not a 100% match I think it still looks good. Let me know if you think I should adjust it in any way

Reply:
Sounds good! I just tested the error pop-up and it looks good, but I wonder if it would be more visible if we move it up (either to the top of the page, or somewhere around the request form) and make it a bit larger? Also, this makes me think of the error page. Do you think we need one and should we ask Kayla to design one? 
<img width="1278" alt="Screen Shot 2024-02-27 at 7 14 23 PM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/60ed51c4-fd7b-4d12-9bf6-94e897921d6d">

Reply:
I updated it to move it to the top with a built-in `position` prop, and I tried to make it bigger by adding a border using a built-in `containerStyle` prop
It doesn't look great, but looks decent. I could make the border our secondary brand red. I do realize that the toast itself has a border radius which is creating a gap between the border and the toast container - does it also need padding to address this?
<img width="1379" alt="Screenshot 2024-02-28 at 10 23 21 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/92ab15f1-dee9-4044-ad1a-7086275099ad">

Reply:
Sounds good, but I would say that just moving it to the top might be enough if we can't really style the toast container.